src/index.html

```diff
@@ -11,12 +11,113 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="stylesheet"
+      href="../../reset/reset.css"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
```

Comment:
why do you need so many types of this font? Import only those you use at this task

Reply:
Still not fixed